src/Data/Utils.purs

```diff
@@ -0,0 +1,31 @@
+module Conduit.Data.Utils where
+
+import Prelude
+
+import Data.Argonaut.Core (Json)
+import Data.Argonaut.Decode (decodeJson)
+import Data.Argonaut.Decode.Struct.Tolerant ((.::))
+import Data.Argonaut.Decode.Struct.Tolerant (class DecodeJson) as Tol
```

Comment:
Could we standardize on importing non-operators from `Data.Decode.Struct.Tolerant` as:

```suggestion
import Data.Argonaut.Decode.Struct.Tolerant as Tolerant
```

...just for uniformity's sake?